pyintegration/deephaven2/numpy.py

```diff
@@ -29,23 +30,27 @@ def freeze_table(table: Table) -> Table:
 
 
 def _to_column_name(name: str) -> str:
-    """ Transforms the given name string into a valid table column name. """
-    tmp_name = re.sub("\W+", " ", str(name)).strip()
-    return re.sub("\s+", "_", tmp_name)
+    """Transforms the given name string into a valid table column name."""
+    tmp_name = re.sub(r"\W+", " ", str(name)).strip()
+    return re.sub(r"\s+", "_", tmp_name)
```

Comment:
Are these "r" values needed when used with `re`?  Is this creating two regular expressions (slow)?